src/coloring/edgeColoring.cpp

```
@@ -29,112 +33,59 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-    Identifiers<int64_t> ids;
-    for (const auto &e : edges) {
-        ids += e.source;
-        ids += e.target;
-    }
-
-    for (const auto id : ids) {
-        auto v = add_vertex(graph);
-        id_to_V.insert(std::make_pair(id, v));
-        V_to_id.insert(std::make_pair(v, id));
-    }
+edgeColoring(const pgrouting::UndirectedGraph g) {
```

Cppcheck (2.20.0)

[performance] Function parameter 'g' should be passed by const reference.

(passedByValue)